lib/cog/command/command_resolver.ex

```diff
@@ -0,0 +1,56 @@
+defmodule Cog.Command.CommandResolver do
+
+  alias Cog.Repo
+  alias Cog.Queries.Command
+  alias Cog.Queries.Alias
+  alias Piper.Command.SemanticError
+
+  def find_bundle(name) when is_binary(name) do
+    if String.contains?(name, ":") do
+      :identity
+    else
+      case Repo.all(Command.bundle_for(name)) do
```

Comment:
If memory serves, returning a `SemanticError` from the old `BundleResolver` with a reason of `:ambiguous_command` would generate such an error message. Based on what I've read so far I believe we preserve the behavior.

As a reasonably helpful and not entirely difficult first step we could send a warning message to the user whenever they execute an alias which shadows a bundle.